pkg/defaults/defaults.go

```diff
@@ -48,7 +48,7 @@ const (
 	DefaultRegistryPort = 5000
 
 	//tags, versions, repos
-	DefaultEVETag               = "0.0.0-master-efa32c66" //DefaultEVETag tag for EVE image
+	DefaultEVETag               = "0.0.0-master-103bd43f" //DefaultEVETag tag for EVE image
```

Comment:
Please make it a stable eve version. Always. 
For tests we may use non-stable using eve.tag

Reply:
Done